finagle-core/src/test/scala/com/twitter/finagle/client/StackClientTest.scala

```diff
@@ -397,4 +398,16 @@ class StackClientTest extends FunSuite
     val client3: StackClient[String, String] =
       client.configured[param.Label]((param.Label("foo"), param.Label.param))
   }
+
+  test("ServiceFactorySocketAddress bypasses transporter") {
```

Comment:
can we either verify that the transporter isn't called, or change the name of this test?